packages/network/networkNames.md

```diff
@@ -0,0 +1,70 @@
+# Network Names
+
+Though networks have proper full names, we sometimes need to refer to a chain by a name slug.
+
+A name slug is the name, sometimes simplified, and in kebab-case. This maximizes compatibility in the event that you want to use chain names as url paths, or filenames.
+
+You can use this file as a reference to look up the correct network name slug a network's full name or its chain ID.
+
+| Network Name | Chain ID | Name Slug |
+| --- | --- | --- |
+| Ethereum | 1 | mainnet |
+| Ropsten | 3 | ropsten |
+| Rinkeby | 4 | rinkeby |
+| Goerli | 5 | goerli |
```

Comment:
This one is deprecated. Please filter out deprecated networks.. or mark them as ~deprecated~.

Reply:
done!